tests/adr/test_adr0001_setup_py_packaging.py

```diff
@@ -0,0 +1,37 @@
+import io
+import os
+import subprocess
+
+import pytest
+
+ROOT_DIR = os.path.realpath(os.path.join(__file__, '..', '..', '..'))
+
+
+@pytest.fixture(scope='module')
+def build_source_distribution():
+    os.chdir(ROOT_DIR)
+    subprocess.check_output(['python', 'setup.py', 'egg_info'])
+    yield None
+
+
+def test_requirement1_controller_ironcar_code_is_present(build_source_distribution):
+    egg_info_dir = 'controller_ironcar_octonomous.egg-info'
+    source_list_file = os.path.join(ROOT_DIR, egg_info_dir, 'SOURCES.txt')
+    with io.open(source_list_file) as f:
+        sources = f.read()
+        assert 'controller_ironcar/' in sources, 'content {file} : {sources}'.format(
+            file=source_list_file, sources=sources)
+
+
+def test_requirement2_default_autopilot_is_present(build_source_distribution):
+    egg_info_dir = 'controller_ironcar_octonomous.egg-info'
+    source_list_file = os.path.join(ROOT_DIR, egg_info_dir, 'SOURCES.txt')
+    with io.open(source_list_file) as f:
+        sources = f.read()
+        assert 'controller_ironcar/resources/autopilots/autopilot_500k.hdf5' in sources, 'content {file} : {sources}'.format(
```

Comment:
cf la remarque ci dessus, je sais pas quel est ce modèle... On teste quoi du coup ?

Reply:
Si je retire le modèle par défaut, le requirement 2 : default autopilot is available in the source distribution saute. Donc le test disparait.